examples/enable-operator-and-auto-instrumentation/spring-petclinic/update-demo.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
I'd move this up a directory, partly so it's not attempted to be used as a manifest

Reply:
Done